### PR TITLE
Organise and clean

### PR DESCRIPTION
**Pull request by JoshuaMarden**
Created an infrastructure directory to put IaS code into. 

Update:

Accidentally used the branch to create infrastructure..

Made a script to auto-dockerise and push containers to AWS.

Added a workflow to automate this after every push to git (IF tests are passed), and corresponding badge in README.

### CODE REVIEW

**Comment by JoshuaMarden** [comment]
Accidentally used the branch to create infrastructure. 

Made a script to auto-dockerise and push containers to AWS.

Added a workflow to automate this after every push to git, and corresponding badge in README.

File: README.md [hunk 1]
```diff
@@ -46,7 +56,7 @@ If you are having trouble using our app please open a ticket we'll get back to y
 
 ## Authors and Acknowledgment
 
-If people have pushed to this repo, they are authors!
+If people have pushed to this repo, they are authors.
```
Comment (HighestAuto) [inline README.md:59]: Nice change, I think it is a nice change in tone

File: README.md [hunk 2]
```diff
@@ -20,6 +29,7 @@ Run the following commands:
 - `python3 -m venv .venv`
 - `bash setup.sh` ! we might want a simple setup script !
```
Comment (HighestAuto) [inline README.md:30]: What does the ! ... ! to do

**Review by HighestAuto — APPROVED** [review]
Looks Good, great that the CI/CD pipeline works very well

**Pull request merged**